requirements/edx/github.txt

```diff
@@ -75,9 +75,9 @@ git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
 git+https://github.com/edx/edx-ora2.git@1.4.6#egg=ora2==1.4.6
--e git+https://github.com/edx/edx-submissions.git@2.0.0#egg=edx-submissions==2.0.0
+git+https://github.com/edx/edx-submissions.git@2.0.1#egg=edx-submissions==2.0.1
```

Comment:
I'm not sure if https://openedx.atlassian.net/browse/EDUCATOR-781 will hit before you get this merged, but you might want to keep an eye on it. With the build in PyPI now this may move to base.txt and cause some weird merge / conflict versions.